checks/net.go

```diff
@@ -150,3 +149,35 @@ func formatType(f tracer.ConnectionType) model.ConnectionType {
 		return -1
 	}
 }
+
+func batchConnections(cfg *config.AgentConfig, groupID int32, cxs []*model.Connection) []model.MessageBody {
+	batches := make([]model.MessageBody, 0)
```

Comment:
Isn't it that `batches := make([]model.MessageBody, 0, groupSize)`? No big deal anyway.